packages/components/src/Notification/ToastNotification/subcomponents/ToastNotificationManager/ToastNotificationManager.spec.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react"
+import { act, screen } from "@testing-library/react"
+import {
+  addToastNotification,
+  removeToastNotification,
+  clearToastNotifications,
+} from "."
+
+describe("ToastNotificationsManager", () => {
+  beforeEach(() => {
+    act(() => {
+      clearToastNotifications()
+    })
+  })
+
+  describe("addToastNotification", () => {
+    it('creates a single "status" container for notifications', async () => {
+      await act(async () => {
```

Comment:
Noting here that since the change to `createRoot` this now throw a test warning about the use of act:

![Screenshot 2023-10-30 at 12 16 52 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/31f28d94-7fe1-437e-b5ec-5d2b3f8488f3)

I haven't figured a way to fix this, though I suspect that it may require a bit more of a refactor. If there any thoughts, would welcome input :)

Reply:
I tried to look into it, and tried multiple things and failed. 
- Wrap `expect`s in `await waitFor`
- Use `createPortal` within the util (fails because it only works inside a JSX.Element)
- Wrap `render` in `act` (sync and async)
- Wrap the util call within `ToastNotification` in a `useEffect`

I believe it is likely caused by `createRoot.render` being called conditionally (it waits for a portal to first be created) which has caused confusion in testing. We'll just need to live with this for now and hope it doesn't cause weird problems.

Reply:
Thanks for validating, I tried a few of those so good to know it wasn't just me 😆 It does feel like this is linked to that. I figure we can keep it as is and come back to this as part of a much needed refactor when we have capacity / if it causes an issue 👍